src/generate-changelog.ts

```diff
@@ -1,21 +1,26 @@
 import * as core from '@actions/core'
-import {graphql} from '@octokit/graphql'
+import { graphql } from '@octokit/graphql'
 import shell from 'shelljs'
 
 // Regexes to find the changelog contents within a PR.
-const FIXES_REGEX = /^CHANGELOG-FIXES:(.*)/gm
-const NEW_REGEX = /^CHANGELOG-NEW:(.*)/gm
-const TEAMS_SPECIFIC_REGEX = /^TEAMS-SPECIFIC-CHANGES:(.*)/gm
+const NEW_FEATURE_REGEX = /^CHANGELOG-NEW-FEATURE:(.*)/gm
```

Comment:
optional nit: consider writing the `NEW_FEATURE_REGEX` as an OR between `CHANGELOG-NEW-FEATURE` and `CHANGELOG-NEW` to avoid defining two separate regexes and calling `parseMatchesFromDescription` an additonal time

Reply:
I think I prefer this method, since it's easier to clean up in a few weeks once no new PRs are using the old prefixes.